tools/bundle-builder/subrunner/index.js

```diff
@@ -91,6 +91,9 @@ function runTaskOnPackages(task, packages) {
 
       if (packageDir) {
         runComponentTask(packageDir, task, function(err) {
+          if (err) {
+            process.exit();
```

Comment:
When running tasks on all components within the monorepo, exit if a failure occurs. `runComponentTask` and gulp itself will log the error already, so we don't need to print it again here.